tests/bdd/bdd_test.go

```diff
@@ -389,12 +392,79 @@ var _ = Describe("BDD on chaos-operator", func() {
 		})
 	})
 
+	Context("Validate via Chaos-Operator Logs", func() {
+
+		It("Should Generate Operator logs", func() {
+			pods, err := client.CoreV1().Pods("litmus").List(metav1.ListOptions{
+				LabelSelector: fmt.Sprintf("%v=%v", "name", "chaos-operator"),
+			})
+			Expect(err).To(BeNil())
+
+			if len(pods.Items) > 1 {
+				klog.Infof("Multiple Chaos-Operator Pods found")
+				return
+			}
+			if len(pods.Items) < 1 {
+				klog.Infof("Unable to find Chaos-Operator Pod")
+				return
+			}
+
+			podName := pods.Items[0].Name
+			Expect(podName).To(
+				Not(BeEmpty()),
+				"Unable to get the operator pod name",
+			)
+
+			klog.Infof("Got Pod Name: %v\n", podName)
+
+			podLogOpts := v1.PodLogOptions{}
+
+			req := client.CoreV1().Pods("litmus").GetLogs(podName, &podLogOpts)
+
+			podLogs, err := req.Stream()
+			Expect(err).To(BeNil())
+
+			defer podLogs.Close()
+
+			buf := new(bytes.Buffer)
+			_, err = io.Copy(buf, podLogs)
+			Expect(err).To(BeNil())
+
+			str := buf.String()
+
+			klog.Infof("Chaos Operator Logs:\n%v\n", str)
+
+		})
+	})
+
 })
 
 //Deleting all unused resources
 var _ = AfterSuite(func() {
 
-	By("Deleting all CRDs")
-	crdDeletion := exec.Command("kubectl", "delete", "-f", "../../deploy/chaos_crds.yaml").Run()
-	Expect(crdDeletion).To(BeNil())
+	//Deleting ChaosExperiments
+	By("Deleting ChaosExperiments")
+	err := exec.Command("kubectl", "delete", "chaosexperiments", "--all", "-n", "litmus").Run()
+	Expect(err).To(BeNil())
+
+	//Deleting ChaosEngines
+	By("Deleting ChaosEngines")
+	err = exec.Command("kubectl", "delete", "chaosengine", "--all", "-n", "litmus").Run()
+	Expect(err).To(BeNil())
+
+	//Deleting CRD's
+	By("deleting chaosengine crd")
+	err = exec.Command("kubectl", "delete", "-f", "../../deploy/chaos_crds.yaml").Run()
+	Expect(err).To(BeNil())
+
+	//Deleting rbacs
+	By("Deleting RBAC's")
+	err = exec.Command("kubectl", "delete", "-f", "../../deploy/rbac.yaml").Run()
+	Expect(err).To(BeNil())
+
+	//Deleting Chaos-Operator
+	By("Deleting operator")
+	err = exec.Command("kubectl", "delete", "-f", "../../deploy/operator.yaml").Run()
+	Expect(err).To(BeNil())
```

Comment:
Remove it in reverse way like: 
1 - operator.yaml 
2 - rbac.yaml
3 - chaos_crds.yaml

Reply:
Fixed @chandankumar4

Reply:
Take a look now, `rbac.yaml` was deleting the namespace `litmus`, thats why operator deletion was failing